docs/docker.md

```diff
@@ -1,10 +1,46 @@
 # Starbase in Docker
 
-## Running Starbase - Docker
-
 Starbase can also be run inside of Docker, minimizing the need to locally
 install node and yarn.
 
+## Execution Flow Diagram
+
+```mermaid
```

Comment:
I'm a newbie with mermaid. How does one go about visualizing the diagram? Does starbase need a new dependency?

Reply:
Same here.  I went and viewed it in Erich's branch and it looks like it renders nicely:  https://github.com/JupiterOne/starbase/blob/erichs-patch-1/docs/docker.md